src/asmcnc/apps/wifi_app/screen_wifi.py

```diff
@@ -666,38 +674,29 @@ def update_strings(self):
         self.password_label.text = self.l.get_bold("Password")
         self.country_label.text = self.l.get_bold("Country")
         self.connect_button.text = self.l.get_str("Connect")
-        self.custom_ssid_button.text = self.l.get_str("Other network")
+        self.custom_ssid_input()
         self.custom_network_name.hint_text = self.l.get_str("Enter network name")
 
         self.update_font_size(self.country_label)
-        self.update_font_size(self.custom_network_name)
-        self.update_button_font_size(self.connect_button, False)
-        self.update_button_font_size(self.custom_ssid_button, True)
+        self.update_hint_font_size(self.custom_network_name)
+        self.update_button_font_size(self.connect_button, 28, 10)
+        self.update_button_font_size(self.custom_ssid_button, 20, 20)
 
     def update_font_size(self, value):
         if len(value.text) < 8:
             value.font_size = self.default_font_size
         elif len(value.text) > 7: 
             value.font_size = self.default_font_size - 2
 
-        try:
-            if value.hint_text:
-                if len(value.hint_text) < 20:
-                    value.font_size = value.font_size
-                elif len(value.hint_text) > 19:
-                    value.font_size = self.default_font_size - 3
-        except:
-            pass
-
-    def update_button_font_size(self, value, small_button):
-        if small_button:
-            value.font_size = 20
-            if len(value.text) > 20:
-                value.font_size = 19
-        else:
-            value.font_size = 28
-            if len(value.text) > 10:
-                value.font_size = 19
+    def update_hint_font_size(self, value):
+        if value.hint_text:
+            if len(value.hint_text) > 22:

```

Comment:
this is a different length to the previous function, is that deliberate and have you re-tested?